docs/user-stories.md

```diff
@@ -0,0 +1,9 @@
+## Glottie Chat User Stories
+
+1. As a researcher whose native language is not English, I want to communicate with peer colleagues in a chat-room where all messages are translated automatically, so that I don't have to fight with English grammar and correct word usage.
+2. As a chat service user, I want to obtain a unique account, so that my personal identity couldn't be forged.
+3. As an authenticated user, I need to be able to create new chat-rooms dedicated to cerain topics and invite other fellow researchers to join them.
```

Comment:
2. Maybe it's too low-level for user stories.

Reply:
> @jeinygroove 2. Maybe it's too low-level for user stories.

Yes, it's just a curiosity question about the app at all)))

Reply:
1. Motivation for this might be to have private discussions within a group.
2. Here I was thinking something along the lines of sending an email invitations to fellow colleagues to join a discussion room. If they already have an account, they can use it, otherwise, the link provided will allow them to join the chat room just for a session (like Zoom?).